edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -79,12 +79,12 @@ <h2 id="acessible-error-title">{% trans "Cannot Start Proctored Exam" %}</h2>
 </div>
 
 <div class="footer-sequence border-b-0 padding-b-0">
-  {% if not is_sample_attempt %}
-  <p class="proctored-exam-instruction">
+  {% if not is_sample_attempt and allow_unproctored %}
+    <p class="proctored-exam-instruction">
       {% trans "Don't want to take this exam with online proctoring?" %}<a class="proctored-decline-exam" href='#' data-action="decline" data-exam-id="{{exam_id}}" data-change-state-url="{{change_state_url}}">
         {% trans "Take this exam as an open exam instead." %}
```

Comment:
@catong I discussed this with @dianakhuang and our conclusion is that these represent two distinct use cases. The other button is shown when you haven't yet chosen whether to choose proctored or 'unproctored':

![image](https://cloud.githubusercontent.com/assets/5985072/19821876/b412e7b6-9d2e-11e6-9791-3d5a76a24a2e.png)

Once you have chosen proctored, you are presented at the bottom of the instructions with the link we are discussing here to change your mind:

![image](https://cloud.githubusercontent.com/assets/5985072/19821787/479864c6-9d2e-11e6-9625-20859c607b23.png)

The UI is quite different so it would be hard to share the logic. However, I'm happy to tweak the wording to make the language more consistent as you see fit.